Bottom: 3bfa752ac206f3d41cab7184bb78c446a7210d3b
Top:    7adb9e874acaa19a22e0a4b3a351672617f52960
Author: Ian Kent <raven@themaw.net>
Date:   2013-05-13 10:06:42 +0800

autofs-5.0.7 - make dump maps report nulled entries

The dumpmaps function was intended for users to find out how the configured
mounts would be seen by automount. It does limited amount of checking and in
particular doesn't report the fact that duplicate indirect map sources will
be ignored at runtime.


---

diff --git a/lib/master.c b/lib/master.c
index 2bb62fe..7663d57 100644
--- a/lib/master.c
+++ b/lib/master.c
@@ -1319,6 +1319,13 @@ int master_show_mounts(struct master *master)
 		ap = this->ap;
 
 		printf("\nMount point: %s\n", ap->path);
+
+		/* Report nulled nulled entries */
+		if (cache_lookup_distinct(master->nc, path)) {
+			printf("\nmap entry %s nulled by previous entry\n",
+			       ap->path);
+		}
+
 		printf("\nsource(s):\n");
 
 		/*
